js/init.js

```diff
@@ -1 +1,51 @@
-//Select branch please
\ No newline at end of file
+const API = 'https://jsonplaceholder.typicode.com/users/';
+let promiseArray = [];
+let dataFromApi = [];
+
+
+//XHR
+for (let i = 1; i <= 10; i++) {
+    promiseArray[i] = new Promise((resolve, reject) => {
```

Comment:
```suggestion
    promiseArray[i-1] = new Promise((resolve, reject) => {
```
чтобы и первый элемент присутствовал:)

Reply:
Точно! Вы на GB сказалм что-то про высшую функцию ("Еще и функцию высшего порядка применяете - за это отдельный лайк"). 

Что вы имели ввиду? Я просто не совсем понял =))

Reply:
Всмысле про какую функцию вы говорите?